README.md

```diff
@@ -50,9 +50,6 @@ To learn more about the programming language, visit [swift.org](https://swift.or
 
 - [Contributing to Swift](#contributing-to-swift)
 - [Getting Started](#getting-started)
-  - [System Requirements](#system-requirements)
-  - [Getting Sources for Swift and Related Projects](#getting-sources-for-swift-and-related-projects)
-  - [Building Swift](#building-swift)
   - [Swift Toolchains](#swift-toolchains)
   - [Build Failures](#build-failures)
 - [Testing Swift](#testing-swift)
```

Comment:
Please make sure to update the table of contents in sync with the content changes; I believe the "Build Dependencies" section, for example, is gone now.